pipeline-runner/R/gem2s-6-prepare_experiment.R

```diff
@@ -188,6 +187,26 @@ prepare_experiment <- function(input, pipeline_config) {
     )
   )
 
+  mitochondrial_config_to_duplicate <- list(
+    auto = "true",
+    filterSettings = list(
+      method = "absolute_threshold",
+      methodSettings = list(absolute_threshold = list(maxFraction = 0.1, binStep = 0.05))
+    )
+  )
+
+  classifier_config_to_duplicate <- list(
+    enabled = tolower(as.character(!any_filtered)), # emptyDrops results not present
+    auto = "true",
+    filterSettings = list(FDR = 0.01)
+  )
+
+  samples <- scdata$samples
```

Comment:
This should be either inside the duplicate_config_per_sample function, or it should be used for the add_custom_config_per_sample function as well.

Reply:
the problem is that in `add_custom_config_per_sample` we use scdata for more things than just getting the samples, while in `duplicate_config_per_sample` we don't.